projects/extension/src/background/ConnectionManagerWithHealth.ts

```diff
@@ -427,6 +428,10 @@ export class ConnectionManagerWithHealth<SandboxId> {
                   break
                 }
               }
+            } else if (jsonRpcMessage.method === "chain_newHead") {
+              chain.bestBlockHeight = parseInt(
+                jsonRpcMessage.params.result.number,
+              )
```

Comment:
```suggestion
```

Reply:
What is the change that is suggested here?

Reply:
Well, to remove this entirely. It was your first approach, which wasn't good.